smartsim/_core/control/controller.py

```diff
@@ -487,15 +515,22 @@ def _prep_entity_client_env(self, entity: Model) -> None:
                         "Co-located was configured for both TCP/IP and UDS"
                     )
                 if port:
-                    client_env["SSDB"] = f"127.0.0.1:{str(port)}"
+                    client_env[f"SSDB{db_name_colo}"] = f"127.0.0.1:{str(port)}"
                 elif socket:
-                    client_env["SSDB"] = f"unix://{socket}"
+                    client_env[f"SSDB{db_name_colo}"] = f"unix://{socket}"
                 else:
                     raise SSInternalError(
                         "Colocated database was not configured for either TCP or UDS"
                     )
         entity.run_settings.update_env(client_env)
 
+    def set_shard_env_var(self, num_shards):
+        self.nums = num_shards
```

Comment:
Is `nums` a good var name?

These two methods set/get_* don't seem to actually touch an env var and are perhaps inappropriately named, too